package.json

```diff
@@ -47,6 +47,6 @@
   },
   "engines": {
     "npm": ">10.0.0 <11.0.0",
-    "node": ">=18.0.0 <19.0.0"
+    "node": ">=20.0.0 <21.0.0"
```

Comment:
Can I ask why it's different here between this version and the [theme](https://github.com/greenpeace/planet4-master-theme/pull/2504/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R56)?